src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -234,6 +234,7 @@ class LeVADocumentUseCase extends DocGenUseCase {
         if (!sections) {
             throw new RuntimeException("Error: unable to create ${documentType}. Could not obtain document chapter data from Jira.")
         }
+        def notDoneIssues = this.getNotDoneIssues(sections)
```

Comment:
@hrcornejo can we improve the naming? What about something like `getDocumentationChaptersNotDone` and the references to `chaptersNotDone`?

Reply:
@hrcornejo resolved